src/main/scala/org/broadinstitute/dsde/vault/services/analysis/UpdateServiceHandler.scala

```diff
@@ -94,7 +94,7 @@ case class UpdateServiceHandler(requestContext: RequestContext, dmService: Actor
         case true =>
           if (fileCount == bossObjects.size) {
             log.debug("'X-Force-Location' Analysis update complete")
-            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, analysis.files))
+            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, Option(providedFiles)))
```

Comment:
OK, let's go through the reasoning of the responses.

UBam Ingest, normal case: return presigned PUT URLs so the caller can upload

UBam Ingest, Force case: return original URLs so the caller can upload

Are these correct:

Analysis Update, normal case: return presigned PUT URLs so the caller can upload ?

Analysis Update, Force case: return original URLs so the caller can upload ?